dev-workflow-ui/webContent/resources/css/workflow-ui.css

```diff
@@ -245,3 +245,36 @@ body .custom-accordion.ui-accordion .ui-accordion-header:first-child {
   padding-top: 0;
   padding-bottom: 0.75em;
 }
+
+/* Notifications */
+.notification-badge {
+  position:absolute;
+  bottom:30px;
+  font-weight:bold;
+  left:20px;
+  font-size:13px;
+  width:22px;
+  text-align:center;
+  padding:2px;
+  height:20px;
+  color:white;
+  border-radius:10px;
+  background-color:var(--ivy-primary-color);
+}
+.notification-button.si {
+  font-size: 1.2em;
+}
+.notification-container {
+  padding:10px;
+  padding-left:0px;
+  padding-right:0px;
+  margin-bottom:0.15em;
+}
+#notification-scroll {
```

Comment:
IDs = bad

Reply:
why?

Reply:
IDs are unique and classes are reusable so if we wanted to this notification somewhere else we should use classes.